python/private/pypi/whl_library_targets.bzl

```
@@ -101,8 +102,16 @@ def whl_library_targets(
         srcs_exclude = [],
         tags = [],
         filegroups = {
-            DIST_INFO_LABEL: ["site-packages/*.dist-info/**"],
-            DATA_LABEL: ["data/**"],
+            EXTRACTED_WHEEL_FILES: dict(
+                include = ["**"],
+                exclude = ["*.whl"],
```

I think the `name` attribute is the wheel filename, so we can be precise with the exclude. If we can pass the sdist filename to this macro, then it would be good as well.
```suggestion
                exclude = [name],
```

---

Is this true for sdists? I guess it doesn't matter -- we'd want to exclude the sdist anyways.